packages/frontend-2/lib/settings/composables/menu.ts

```diff
@@ -27,7 +28,7 @@ export const useSettingsMenu = () => {
     },
     security: {
       title: 'Security',
-      disabled: true,
+      component: SettingsWorkspacesSecurity,
       tooltipText: 'SSO, manage permissions, restrict domain access'
```

Comment:
We can remove the tooltip, as it's active now, and add this text as the subheader on the page (maybe just change it to what you can actually do on the page)